### PR TITLE
[type: refactor] add http client in yaml

### PR DESCRIPTION
**Pull request by yu199195**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-bootstrap/src/main/resources/application.yml [hunk 1]
```diff
@@ -86,6 +86,39 @@ shenyu:
...
+  #    allIdleTime: 3000
+  #    readTimeout: 3000
+  #    writeTimeout: 3000
+  #    wiretap: false
+  #    keepAlive: false
+  #    pool:
+  #      type: ELASTIC
+  #      name: proxy
+  #      maxConnections: 16
+  #      acquireTimeout: 45000
```
Comment (AhahaGe) [inline shenyu-bootstrap/src/main/resources/application.yml:104]: maybe need 
/**
         * Time in millis after which the channel will be closed,
         * if NULL there is no max idle time.
         */
        private Long maxIdleTime;

File: shenyu-bootstrap/src/main/resources/application.yml [hunk 2]
```diff
@@ -86,6 +86,39 @@ shenyu:
...
+  #      useInsecureTrustManager: true
+  #      keyStoreType: PKCS12
+  #      keyStorePath: classpath:keystore.p12
+  #      keyStorePassword: 123456
+  #      keyPassword: 123456
+  #      trustedX509Certificates:
+  #      handshakeTimeout:
+  #      closeNotifyFlushTimeout:
+  #      closeNotifyReadTimeout:
+  #      defaultConfigurationType:
```
Comment (AhahaGe) [inline shenyu-bootstrap/src/main/resources/application.yml:121]: maybe need keyStoreProvider

/** 
         * Keystore provider for netty httpClient and webclient.
         */
        private String keyStoreProvider;

File: shenyu-bootstrap/src/main/resources/application.yml (hunk 1 above)
Comment (yu199195) [reply shenyu-bootstrap/src/main/resources/application.yml:104]: ok

**Comment by AhahaGe** [comment]
LGTM

**Review by haibo-duan — APPROVED** [review]

**Review by qicz — APPROVED** [review]

**Pull request merged**